styles.css

```diff
@@ -130,3 +256,125 @@ section {
   display: flex;
   justify-content: space-between;
 }
+
+section {
+  box-sizing: border-box;
+  min-height: 100vh;
+  padding: 0 24px 0 24px;
+  width: 100%;
+}
+
+.snapshoot {
+  width: fit-content;
+  height: fit-content;
+}
+
+.skills {
+  display: flex;
+  justify-content: baseline;
+  gap: 1.2rem;
+}
+
+.text-tonic {
+  width: 295px;
+  height: 76px;
+  padding: 20px 0 12px 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.skills li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  height: 24px;
+  background: #ebebff;
+  text-align: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  padding: 4px 12px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.li-javascript {
+  width: 89px;
+}
+
+.li-css {
+  width: 45px;
+}
+
+.li-html {
+  width: 53px;
+}
+
+/* Languages & Frameworks */
+.lang-frame-skills {
+  background-color: #fff;
+}
+
+.lang-frame-skills_languages {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #000;
+  padding-right: 2.3rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  height: 70px;
+}
+
+.javascript .html5 .css {
+  display: flex;
+  align-items: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+  text-align: center;
+}
+
+.arrow-down {
+  padding: 0.9rem 1.6rem 1rem 1.2rem;
+}
+
+.lang-frame-skills_icons li {
+  display: flex;
+  align-items: center;
+  color: #253858;
+  font-size: 1.5rem;
+  font-weight: 500;
+  line-height: 2rem;
+  background-color: #f7f7f9;
+  margin: 1.2rem 0;
+}
```

Comment:
[OPTIONAL]

Please consider adding the following properties to these items so they can look more like the Figma design: :smiley: 

```
max-width
border-radius
padding
```